### DISCUSSION TITLE
Raggiungere rasperry da Terminal con SSD

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2032] [Hardware & Firmware]
Ciao a tutti, 
sto installando HA con Raspberry PI4 su SSD come da guida.
premetto che non avendo nulla, avevo installato sulla SD docker, configurato il rasperry etc etc raggiungendo tutto da ssh.
da quando ho rimosso la SD e agganciato SSD si è avviato HA ma non raggiungo più il rasperry da ssh.
dove sto sbagliando?
premetto che ora è in rete cablata con DHCP attivo quindi vedo l'indirizzo di HA tranquillamente e lo raggiungo da web.

**Comment by snakuzzo** [comment]
ma hai installato HA OS ?
In tal caso non hai accesso SSH di default. Devi installare addon aggiuntivo

**Comment by Matty1184** [comment]
si, pensavo che il sistema operativo però restasse sono onesto!
quindi cerco l'addon...
info.. meglio HA come OS oppure tenere Rasperry PI e spostare solo il DB di HA?

**Reply by snakuzzo** [reply]
Il db non c'entra niente...e...sono tutte tue scelte in base alle tue conoscenze. Non posso dirti io cosa è meglio. Leggi il nostro articolo su quale installazione può fare al caso tuo

**Comment by Matty1184** [comment]
le ho viste però non riesco a capire come installare Supervised ma utilizzare l'SSD anzichè la micro sd

**Reply by snakuzzo** [reply]
Devi installare Debian su SSD e poi seguire la guida per l'installazione HA Supervised

P.S. cortesemente fai reply sotto l'ultimo messaggio...non creare thread nuovi come stai facendo ora
